# HG changeset patch
# User Chris <cspiel@users.sourceforge.net>
# Date 1438958604 -7200
# Node ID 8387f0170f7b352baea887f77960e8066185a6cd
# Parent  c1bcd2df4e3724f3bb0414d3a5e2e9302be78921
Get the Texinfo documentation to translate again with makeinfo-5.2.

This patch takes thus care of issue #1181678
    https://bugs.launchpad.net/enblend/+bug/1181678

We do not strive for a nicely formatted documentation anymore in the
Stable Branch, because the Development Branch already has a flawless
new documentation *not* based on Texinfo.

diff -r c1bcd2df4e37 -r 8387f0170f7b doc/Makefile.am
--- a/doc/Makefile.am	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/Makefile.am	Fri Aug 07 16:43:24 2015 +0200
@@ -46,7 +46,7 @@
                    -I $(top_builddir) -I $(srcdir) \
                    --css-include=@srcdir@/default.css \
                    $(MAKEINFOHTMLFLAGS)
-export TEXINPUTS=$(top_builddir):$(srcdir)
+export TEXINPUTS = .:$(top_builddir):$(srcdir):
 
 TEXI2DVI = texi2dvi $(TEXI2DVIFLAGS) $(EXTRATEXI2DVIFLAGS)
 
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/auxmac.tex
--- a/doc/auxmac.tex	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/auxmac.tex	Fri Aug 07 16:43:24 2015 +0200
@@ -1,13 +1,6 @@
-\input thumbpdf.sty
-
-
 % Auxilliary Macros.
 %
 % Include this file before texinfo.tex!
 
 \def\mathit#1{\hbox{\it #1}}
 \def\mathrm#1{\ifmmode{\rm #1}\else #1\fi}
-
-% These definitions are required for older versions of texinfo.tex.
-\def\geq{\ifmmode \ge\else $\ge$\fi}
-\def\leq{\ifmmode \le\else $\le$\fi}
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/auxmac.texi
--- a/doc/auxmac.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/auxmac.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -2,16 +2,6 @@
 @c Macro Definitions
 @c
 
-@c redefined commands
-
-@c Get the spacing of dimensions right.
-@ifnottex
-@macro dmn{unit}
-@tie{}\unit\
-@end macro
-@end ifnottex
-
-
 @c extended commands
 
 @c Add a title to a DocBook element.
@@ -80,73 +70,6 @@
 
 @c Operators
 
-@c Generate a nice representation of base^exponent.
-@macro power{base, exponent}
-@ifinfo
-\base\^\exponent\
-@end ifinfo
-@html
-\base\<sup>\exponent\</sup>
-@end html
-@tex
-$\base\^{\exponent\}$%
-@end tex
-@docbook
-\base\<superscript>\exponent\</superscript>
-@end docbook
-@end macro
-
-
-@macro classictimes
-@ifinfo
-x@c gobble following newline  -- The Tricks of a Texinfo Wizard.
-@end ifinfo
-@html
-&times;
-@end html
-@tex
-\\ifmmode\\times\\else$\\times$\\fi% gobble following newline  -- The Tricks of a TeX Wizard.
-@end tex
-@docbook
-&times;
-@end docbook
-@end macro
-
-
-@c Required for older versions of makeinfo.  The definition of @geq
-@c for TeX lives in auxmac.tex.
-@ifnottex
-@macro geq
-@ifinfo
->=@c
-@end ifinfo
-@html
-&ge;
-@end html
-@docbook
-&ge;
-@end docbook
-@end macro
-@end ifnottex
-
-
-@c Required for older versions of makeinfo.  The definition of @leq
-@c for TeX lives in auxmac.tex.
-@ifnottex
-@macro leq
-@ifinfo
-<=@c
-@end ifinfo
-@html
-&le;
-@end html
-@docbook
-&le;
-@end docbook
-@end macro
-@end ifnottex
-
-
 @macro plusminus
 @ifinfo
 +/-@c
@@ -163,62 +86,6 @@
 @end macro
 
 
-@c Special Characters
-
-@macro inlineomega
-@ifinfo
-@math{omega}@c
-@end ifinfo
-@html
-<mathinline xmlns="http://www.w3.org/1998/Math/MathML" display="inline">
-    <mi>&omega;</mi>
-</mathinline>
-@end html
-@tex
-$\\omega$%
-@end tex
-@docbook
-&omega;
-@end docbook
-@end macro
-
-
-@macro inlinesigma
-@ifinfo
-@math{sigma}@c
-@end ifinfo
-@html
-<mathinline xmlns="http://www.w3.org/1998/Math/MathML" display="inline">
-    <mi>&sigma;</mi>
-</mathinline>
-@end html
-@tex
-$\\sigma$%
-@end tex
-@docbook
-&sigma;
-@end docbook
-@end macro
-
-
-@macro inlinexi
-@ifinfo
-@math{xi}@c
-@end ifinfo
-@html
-<mathinline xmlns="http://www.w3.org/1998/Math/MathML" display="inline">
-    <mi>&xi;</mi>
-</mathinline>
-@end html
-@tex
-$\\xi$%
-@end tex
-@docbook
-&xi;
-@end docbook
-@end macro
-
-
 @c Text Fragments
 
 @macro mainpurpose
@@ -236,26 +103,3 @@
 @noindent
 @strong{Summary of influential options}
 @end macro
-
-
-@macro semilog{significant, exponent}
-@ifinfo
-\significant\*10^\exponent\@c
-@end ifinfo
-@html
-<math xmlns="http://www.w3.org/1998/Math/MathML" display="inline">
-    <mn>\significant\</mn>
-    <mo>&times;</mo>
-    <msup>
-        <mn>10</mn>
-        <mn>\exponent\</mn>
-    </msup>
-</math>
-@end html
-@tex
-\\ifmmode\significant\ \\times 10^{\exponent\}\\else$\significant\ \\times 10^{\exponent\}$\\fi%
-@end tex
-@docbook
-\significant\&times;10<superscript>\exponent\</superscript>
-@end docbook
-@end macro
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/bug-reports.texi
--- a/doc/bug-reports.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/bug-reports.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -77,7 +77,7 @@
 @item
 A complete set of input images that will reproduce the bug.  Strive
 for a minimal set of @emph{small}@footnote{Images of a size less than
-1500@classictimes{}1000 pixels qualify as small.} images.
+1500x1000 pixels qualify as small.} images.
 
 @item
 The type of machine you are using, and the operating system name and
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/enblend.info
Binary file doc/enblend.info has changed
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/enblend.texi
--- a/doc/enblend.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/enblend.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -508,9 +508,9 @@
 twice as large as the previous one.  So, the zeroth layer, the
 original image, obviously defines the image at single-pixel scale, the
 first level works at two-pixel scale, and generally, the @math{n}-th
-level contains image data at @power{2, n}-pixel scale.  This is the
+level contains image data at 2^n-pixel scale.  This is the
 reason why an image of
-@math{width}@classictimes{}@/@math{height}@dmn{pixels} cannot be
+@math{width}x@/@math{height}@dmn{pixels} cannot be
 deconstructed into a pyramid of more than
 @ifinfo
 @display
@@ -1194,11 +1194,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{1.2, -38}
+Minimum normalized value: 1.2e-38
 @item
-Epsilon: @semilog{1.2, -7}
+Epsilon: 1.2e-7
 @item
-Maximum finite value: @semilog{3.4, 38}
+Maximum finite value: 3.4e38
 @end itemize
 
 @c IEEE double: 64 bits, n = 53, k = 64 - n - 1 = 10
@@ -1210,11 +1210,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{2.2, -308}
+Minimum normalized value: 2.2e-308
 @item
-Epsilon: @semilog{2.2, -16}
+Epsilon: 2.2e-16
 @item
-Maximum finite value: @semilog{1.8, 308}
+Maximum finite value: 1.8e308
 @end itemize
 @end table
 
@@ -1236,11 +1236,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{9.3, -10}
+Minimum normalized value: 9.3e-10
 @item
-Epsilon: @semilog{2.0, -3}
+Epsilon: 2.0e-3
 @item
-Maximum finite value: @semilog{4.3, 9}
+Maximum finite value: 4.3e9
 @end itemize
 
 @item -f @var{WIDTH}x@var{HEIGHT}
@@ -1250,7 +1250,7 @@
 @cindex canvas size
 @cindex size, canvas
 Ensure that the minimum ``canvas'' size of the output image is at
-least @var{WIDTH}@classictimes{}@/@var{HEIGHT}.  Optionally specify
+least @var{WIDTH}x@/@var{HEIGHT}.  Optionally specify
 the @var{XOFFSET} and @var{YOFFSET}, too.
 
 @pindex nona @r{(Hugin)}
@@ -1572,7 +1572,7 @@
 @value{src::default-coarseness-factor}, this means,
 option@tie{}@option{--coarse-mask} shrinks the overlapping
 @emph{areas} by a factor of
-@math{@value{src::default-coarseness-factor}@classictimes{}@/@value{src::default-coarseness-factor}}.
+@math{@value{src::default-coarseness-factor}x@/@value{src::default-coarseness-factor}}.
 With @var{FACTOR}@tie{}=@tie{}8 the total memory allocated during a
 run of Enblend shrinks approximately by 80% and the maximum amount of
 memory in use at a time is decreased to 60% (Enblend compiled with
@@ -2494,7 +2494,7 @@
 When using this option in conjunction with
 option@tie{}@code{--coarse-mask}=@/@var{FACTOR}, keep in mind that the
 smoothing occurs @emph{after} the overlap regions have been shrunken.
-Thus, blurring affects a @var{FACTOR}@classictimes{}@/@var{FACTOR}
+Thus, blurring affects a @var{FACTOR}x@/@var{FACTOR}
 times larger area in the original images.
 
 Valid range: @var{RADIUS} @geq{} @value{src::minimum-smooth-difference}.
@@ -2606,7 +2606,7 @@
 @rimage{seam-line-visualization}
 
 @caption{Seam-line visualization of a simple overlap.  The
-853@classictimes{}238@dmn{pixel} image is shown at a magification of
+853x238@dmn{pixel} image is shown at a magification of
 100%.}
 
 @shortcaption{Seam-line visualization}
@@ -2618,7 +2618,7 @@
 @rimage{seam-line-visualization, 15cm}
 
 @caption{Seam-line visualization of a simple overlap.  The
-853@classictimes{}238@dmn{pixel} image has been rescaled to a width of
+853x238@dmn{pixel} image has been rescaled to a width of
 approximately 15@dmn{cm}.}
 
 @shortcaption{Seam-line visualization}
@@ -2629,7 +2629,7 @@
 @float Figure,Figure:seam-line-visualization
 @rimage{seam-line-visualization, 15cm}
 
-@caption{Seam-line visualization of a simple overlap.  The 853@classictimes{}238@dmn{pixel} image has been rescaled to a width of approximately 15@dmn{cm}.}
+@caption{Seam-line visualization of a simple overlap.  The 853x238@dmn{pixel} image has been rescaled to a width of approximately 15@dmn{cm}.}
 
 @shortcaption{Seam-line visualization}
 @end float
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/enfuse.info
Binary file doc/enfuse.info has changed
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/enfuse.texi
--- a/doc/enfuse.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/enfuse.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -621,9 +621,9 @@
 twice as large as the previous one.  So, the zeroth layer, the
 original image, obviously defines the image at single-pixel scale, the
 first level works at two-pixel scale, and generally, the @math{n}-th
-level contains image data at @power{2, n}-pixel scale.  This is the
+level contains image data at 2^n-pixel scale.  This is the
 reason why an image of
-@math{width}@classictimes{}@/@math{height}@dmn{pixels} cannot be
+@math{width}x@/@math{height}@dmn{pixels} cannot be
 deconstructed into a pyramid of more than
 @ifinfo
 @display
@@ -1255,11 +1255,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{1.2, -38}
+Minimum normalized value: 1.2e-38
 @item
-Epsilon: @semilog{1.2, -7}
+Epsilon: 1.2e-7
 @item
-Maximum finite value: @semilog{3.4, 38}
+Maximum finite value: 3.4e38
 @end itemize
 
 @c IEEE double: 64 bits, n = 53, k = 64 - n - 1 = 10
@@ -1271,11 +1271,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{2.2, -308}
+Minimum normalized value: 2.2e-308
 @item
-Epsilon: @semilog{2.2, -16}
+Epsilon: 2.2e-16
 @item
-Maximum finite value: @semilog{1.8, 308}
+Maximum finite value: 1.8e308
 @end itemize
 @end table
 
@@ -1297,11 +1297,11 @@
 
 @itemize
 @item
-Minimum normalized value: @semilog{9.3, -10}
+Minimum normalized value: 9.3e-10
 @item
-Epsilon: @semilog{2.0, -3}
+Epsilon: 2.0e-3
 @item
-Maximum finite value: @semilog{4.3, 9}
+Maximum finite value: 4.3e9
 @end itemize
 
 @item -f @var{WIDTH}x@var{HEIGHT}
@@ -1311,7 +1311,7 @@
 @cindex canvas size
 @cindex size, canvas
 Ensure that the minimum ``canvas'' size of the output image is at
-least @var{WIDTH}@classictimes{}@/@var{HEIGHT}.  Optionally specify
+least @var{WIDTH}x@/@var{HEIGHT}.  Optionally specify
 the @var{XOFFSET} and @var{YOFFSET}, too.
 
 @pindex nona @r{(Hugin)}
@@ -1501,8 +1501,8 @@
 @var{LCE-SCALE} is the radius of the Gaussian used in the enhancement
 step, @var{LCE-FACTOR} is the weight factor (``strength'').
 
-@var{enhanced} = (1 + @var{LCE-FACTOR}) @classictimes{} @var{original}
-@minus{} @var{LCE-FACTOR} @classictimes{} Gaussian@/Smooth(@var{original},
+@var{enhanced} = (1 + @var{LCE-FACTOR}) x @var{original}
+@minus{} @var{LCE-FACTOR} x Gaussian@/Smooth(@var{original},
 @var{LCE-SCALE}).
 
 @var{LCE-SCALE} defaults to @value{src::default-lce-scale} pixels and
@@ -1531,7 +1531,7 @@
 @opindex --contrast-window-size
 
 Set the window @var{SIZE} for local contrast analysis.  The window
-will be a square of @var{SIZE}@classictimes{}@/@var{SIZE} pixels.  If
+will be a square of @var{SIZE}x@/@var{SIZE} pixels.  If
 given an even @var{SIZE}, Enfuse will automatically use the next odd
 number.
 
@@ -1585,7 +1585,7 @@
 @opindex --entropy-window-size
 
 Window @var{SIZE} for local entropy analysis.  The window will be a
-square of @var{SIZE}@classictimes{}@/@var{SIZE} pixels.
+square of @var{SIZE}x@/@var{SIZE} pixels.
 
 In the entropy calculation @var{SIZE} values of 3 to 7 yield an
 acceptable compromise of the locality of the information and the
@@ -2433,20 +2433,6 @@
 from 1 to the common height, and @math{i} from 1 to the number of
 input images@tie{}@math{n}.
 
-@macro equationW{}
-@ifnotdocbook
-@ifnottex
-(W)
-@end ifnottex
-@end ifnotdocbook
-@tex
-(W)%
-@end tex
-@docbook
-<xref linkend="equ:pixel-weighting-function"/>
-@end docbook
-@end macro
-
 Enfuse allows for weighting the contribution of each @math{P(i, x, y)}
 to the final @math{Q(x, y)}:
 @ifinfo
@@ -2454,7 +2440,7 @@
 @math{w(P(1, x, y)) * P(1, x, y) +
 ... +
 w(P(n, x, y)) * P(n, x, y)
---> Q(x, y),}@w{                                                  }@equationW{}
+--> Q(x, y),}
 @end display
 @end ifinfo
 @html
@@ -2520,9 +2506,6 @@
                 <mi>y</mi>
             </mfenced>
         </mrow>
-        <mtext>,</mtext>
-        <mspace width="4em"/>
-        <mtext>@equationW{}</mtext>
     </mrow>
 </math>
 @end html
@@ -2530,7 +2513,7 @@
 $$
     w(P(1, x, y)) P(1, x, y) + \ldots + w(P(n, x, y)) P(n, x, y)
     \rightarrow
-    Q(x, y),\hskip4em\hbox{@equationW{}}
+    Q(x, y)
 $$
 @end tex
 @docbook
@@ -2948,7 +2931,7 @@
 be extreme, favoring only a few pixels or even only one pixel in the
 input stack.  Extremes are not typical, however.
 
-Equal weights are another extreme that turns @equationW{} into an
+Equal weights are another extreme that turns the equation into an
 arithmetic average.  This is why we sometimes speak of the ``averaging
 property'' of this weighting algorithm, like smoothing out noise.
 
@@ -2969,7 +2952,7 @@
 wins, this is, gets weight@tie{}one, and all other pixels get the
 weight of zero
 (@uref{http://@/en.wikipedia.org/@/wiki/@/The_@/Winner_@/Takes_@/It_@/All,,``The
-Winner Takes It All.''}).  With @option{--hard-mask} Equation@tie{}@equationW{}
+Winner Takes It All.''}).  With @option{--hard-mask} the equation
 becomes
 @ifinfo
 @display
@@ -3114,8 +3097,8 @@
 
 @noindent
 Note that this ``averaging'' scheme lacks the nice noise-reduction
-property of the weighted average@tie{}@equationW{}, because only a
-single input pixel contributes to the output.
+property of the weighted average, because only a single input pixel
+contributes to the output.
 
 
 @node Single Criterion Fusing
@@ -3562,7 +3545,7 @@
 
 @noindent
 It associates a probability@tie{}@math{p} with each of the @math{n}
-different possible outcomes@tie{}@inlineomega{} of the random
+different possible outcomes@tie{}@math{omega} of the random
 variable@tie{}@math{X}.
 @cindex expectation value
 Based on @math{w}, we define the @dfn{expectation value} or ``First
@@ -4189,7 +4172,7 @@
 @end docbook
 
 @noindent
-The parameter@tie{}@inlinesigma{}, the argument of
+The parameter@tie{}@math{sigma}, the argument of
 option@tie{}@option{--contrast-edge-scale}, is the length scale on which edges
 are detected by @math{g(x, y)}.  We apply the Laplacian operator in
 Cartesian coordinates
@@ -4401,7 +4384,7 @@
 </informalequation>
 @end docbook
 
-where we have used the dimensionless distance@tie{}@inlinexi{} from
+where we have used the dimensionless distance@tie{}@math{xi} from
 the origin
 @ifinfo
 @display
@@ -4556,7 +4539,7 @@
 @float Figure,Figure:laplacian-of-gaussian
 @vimage{laplacian-of-gaussian}
 
-@caption{Laplacian-of-Gaussian function for @inlinesigma{} = 0.5.}
+@caption{Laplacian-of-Gaussian function for @math{sigma} = 0.5.}
 
 @shortcaption{Laplacian-of-Gaussian}
 @end float
@@ -4656,10 +4639,10 @@
 @var{CURVATURE} nor the mode of operation (@acronym{SDev}-only,
 @acronym{LoG}-only, or a blend of both) scales to different image
 sizes.  In practice, this means that if you start with a set of
-reduced size images, say 2808@classictimes{}1872 pixels, carefully
+reduced size images, say 2808x1872 pixels, carefully
 optimize @var{EDGESCALE}, @var{CURVATURE} and so on, and find
 @acronym{LoG}-only the best mode, and then switch to the original
-resolution of 5616@classictimes{}3744 pixels, multiplying (or
+resolution of 5616x3744 pixels, multiplying (or
 dividing) the parameters by four and sticking to @acronym{LoG}-only
 might @emph{not} result in the best fused image.  For best quality,
 perform the parameter optimization and the search for the most
@@ -5080,9 +5063,9 @@
 Images should align well to be suitable for fusion.  However, there is
 no hard mathematical rule what ``well'' means.  The alignment
 requirements for 16@dmn{MPixel} images to yield a sharp
-4"@classictimes{}6" print at 300@dmn{dpi} (``dpi'' means dots per
+4"x6" print at 300@dmn{dpi} (``dpi'' means dots per
 inch) or even for web presentation are relatively low, whereas the
-alignment of 8@dmn{MPixel} images for a 12"@classictimes{}18" print
+alignment of 8@dmn{MPixel} images for a 12"x18" print
 ought to be tight.
 
 @pindex hugin
@@ -5548,7 +5531,7 @@
 
 Let us use an example to illustrate the problem of relating the
 sharpness with the local contrast variations.  Say we use a
-5@classictimes{}5 contrast window.  Moreover, let @code{sharp_edge}
+5x5 contrast window.  Moreover, let @code{sharp_edge}
 and @code{smooth_edge} be two specific configurations:
 
 @example
@@ -5727,7 +5710,7 @@
 Apply the default grayscale projector: @code{average} and throw away
 all edges with a curvature of less than 0.5% and replace the
 @acronym{LoG} data between 0% and 0.5% with @acronym{SDev} data.  Use
-a window of 7@classictimes{}7@dmn{pixel} window to compute the
+a window of 7x7@dmn{pixel} window to compute the
 @acronym{SDev}.
 @end table
 
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/versenblend.texi
--- a/doc/versenblend.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/versenblend.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -1,4 +1,4 @@
-@set UPDATED 10 March 2014
-@set UPDATED-MONTH March 2014
+@set UPDATED 15 June 2015
+@set UPDATED-MONTH June 2015
 @set EDITION 4.1.3
 @set VERSION 4.1.3
diff -r c1bcd2df4e37 -r 8387f0170f7b doc/versenfuse.texi
--- a/doc/versenfuse.texi	Fri Aug 07 16:43:18 2015 +0200
+++ b/doc/versenfuse.texi	Fri Aug 07 16:43:24 2015 +0200
@@ -1,4 +1,4 @@
-@set UPDATED 10 March 2014
-@set UPDATED-MONTH March 2014
+@set UPDATED 15 June 2015
+@set UPDATED-MONTH June 2015
 @set EDITION 4.1.3
 @set VERSION 4.1.3

